src/main/resources/db/migration/V27.0__DDL_create_holiday_table_update_permissions.sql

```diff
@@ -0,0 +1,136 @@
+  date  timestamp    NOT NULL,
+  description varchar(100) NOT NULL,
+  created_at  timestamp    NOT NULL,
+  modified_at timestamp    NOT NULL,
+  deleted     boolean      NOT NULL,
+  CONSTRAINT holiday_country_fk FOREIGN KEY (country_id) REFERENCES country(id)
+)
+;
+
+UPDATE config SET "role"='ROLE_ADMIN', permissions='[
+  {"entity": "employees", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "contracts", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "assignments", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "clients", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "companies", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "projects", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "project-types", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "roles", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "seniorities", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "technologies", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "tenants", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "leave-types", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "holidays", "actions": ["create", "read", "update", "delete"]},
+  {"entity": "countries", "actions": ["create", "read", "update", "delete"]}
+]'::json::json, menu='[
+  {
+    "name": "Employees",
+    "href": "/#/employees",
+    "icon": "employees",
+    "key": 1
+  },
+  {
+    "name": "Contracts",
+    "href": "/#/contracts",
+    "icon": "contracts",
+    "key": 2
+  },
+  {
+    "name": "Assignments",
+    "href": "/#/assignments",
+    "icon": "assignments",
+    "key": 3
+  },
+  {
+    "name": "Clients",
+    "href": "/#/clients",
+    "icon": "clients",
+    "key": 4
+  },
+  {
+    "name": "Settings",
+    "icon": "settings",
+    "key": 5,
+    "submenu": [
+      {
+        "name": "Companies",
+        "href": "/#/companies",
+        "key": 51
+      },
+      {
+        "name": "Countries",
+        "href": "/#/countries",
+        "key": 52
+      },
+      {
+        "name": "Holidays",
+        "href": "/#/holidays",
+        "key": 53
+      },
+      {
+        "name": "Leave Types",
+        "href": "/#/leave-types",
+        "key": 54
+      },
+      {
+        "name": "Projects",
+        "href": "/#/projects",
+        "key": 55
+      },
+      {
+        "name": "Project types",
+        "href": "/#/project-types",
+        "key": 56
+      },
+      {
+        "name": "Roles",
+        "href": "/#/roles",
+        "key": 57
+      },
+      {
+        "name": "Seniorities",
+        "href": "/#/seniorities",
+        "key": 58
+      },
+      {
+        "name": "Technologies",
+        "href": "/#/technologies",
+        "key": 59
+      },
+      {
+        "name": "Tenants",
+        "href": "/#/tenants",
+        "key": 511
```

Comment:
:thinking: ?

Reply:
how does the numbering follow after key 59?